docs/conf/prague/2019/news/welcome-getready.rst

```diff
@@ -0,0 +1,53 @@
+:template: {{year}}/generic.html
+
+.. post:: Sep 10, 2019
+   :tags: practical info, prague-2019
+
+Welcome to Write the Docs Prague 2019
+=====================================
+
+Yet another edition of Write the Docs Prague nearly upon us! We're excited to see you all in just a few days.
+
+Below you'll find all the practical information you need to navigate the event and activities.
+
+Be prepared
+-----------
+
+* **Read the Welcome Wagon Guide.** Our `Welcome Wagon Guide <https://www.writethedocs.org/conf/prague/2019/welcome-wagon/>`_ is packed full of information on how to get the most out of the conference. There will also be a few Welcome Wagon events that will help you learn more about the conference.
```

Comment:
Are we still calling it this? I thought the name got changed?

Reply:
We didn't change the name, we just added some info in the printed signs and schedules to clarify what the activities entail (tour, meet and greet). We said that we'll discuss the name on a global level at some point, but nothing that affects this year.